tensor_genn/algorithms/relu_ann.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import random
+
+import tensorflow as tf
+from pygenn import genn_model, genn_wrapper
+
+class ReLUANN():
+    def __init__(self,neuron_resting_voltage=-60.0,neuron_threshold_voltage=-55.0,
+        membrane_capacitance=1.0, model_timestep=1.0,):
+        self.Vres = neuron_resting_voltage
+        self.Vthr = neuron_threshold_voltage
+        self.Cm = membrane_capacitance
+        self.timestep = model_timestep
+
+    def convert(self, tf_model):
+        # create custom classes
```

Comment:
This function probably needs to check that tf_model is a keras ``Sequential`` as, otherwise I don't think it works. Otherwise throw a ``NotImplementedError``